### ISSUE TITLE
[BUG] Too many records records on DB

### ISSUE THREAD

**Issue by pedrobsm** [StyraHem/ShellyForHASS#471]
## Environment

- ShellyForHASS release with the issue: 2.1 beta
- Last working ShellyForHASS release (if known): 
- Home Assistant Core release with the issue: 
- Operating environment (Home Assistant/Supervised/Docker/venv): Hassio

## Describe the bug
Since I have a Shelly 1PM controlling my heating system, my HA DB grew up exponentially.
I noticed that when power consumption varies, the rate at which new state records are stored on the database goes up to 200-300 per second.
Then I noticed it happends to every current consumption sensor, on shelly 1PM and Shelly 2.5.

## Steps to Reproduce
Just query the DB records for time periods when power consumption is not steady.

## Expected behavior
 Shouldn't the record rate be constantor at least limited?

**Comment by asilgalis** [comment]
Any update on this?
I am having the same issue. I've had to downgrade shelly devices to 1.8.3 in order to have steady records in HA.

**Comment by pedrobsm** [comment]
Not really. I had to remove most of the sensor from the recorder manually on the configuration file. Still, I'me geting 10x more events than before. :/

**Comment by asilgalis** [comment]
@hakana has this issue been looked at?

**Comment by hakana** [comment]
Ii think this is the way HA working. You can change number of decimals to have lower logging. You can also exclude sensors for recorder in HA.
